Nightly-Tests/Sync-Tests/createCluster.sh

```diff
@@ -0,0 +1,86 @@
+set -x
+
+trap ctrl_c INT
+
+function ctrl_c() {
+    echo -e "\nExit called by user"
+    for pod in $(kubectl get pods -o jsonpath='{.items[*].metadata.name}');do
+        echo "Pulling node logs..." 
+        kubectl logs $pod > ./SyncOutput/$DATE/$(kubectl get pod $pod -o jsonpath='{.metadata.labels.nodenum}').log
+    done
+
+    timeout 10 tiab/teardown_cluster.py -t $UUID -n $K8S_NAMESPACE
+    deactivate
+    exit
+}
+
+if [ "$#" -ne 1 ]; then
+    echo "Please specify an image to test"
+    exit 1
+fi
+
+set -x
+
+echo "Downloading apt requirments "
+sed 's/#.*//' requirements.txt | xargs apt-get install -y
+
+UUID="$(uuidgen)"
+K8S_NAMESPACE=$(kubectl config get-contexts $(kubectl config current-context) | tail -n+2 | awk '{print $5}')
+base_dir=$(pwd)
+IMAGE=${SYNC_IMAGE:-"iotacafe/iri-dev"}
+DATE=$(date '+%Y-%m-%d')
+
+if [ ! -d tiab ]; then
+  git clone --depth 1 https://github.com/iotaledger/tiab tiab
+fi
+ 
+virtualenv -p python2 venv
```

Comment:
python 2?

Reply:
Yeah, TIAB is currently only compatible with Python2 :/. There is an issue open on the repo already to port it to 3